docs/dev_guide/code_standards.rst

```diff
@@ -52,24 +52,37 @@ Coding Style/Conventions
 Formatting
 ==========
 
-We enforce a minimum level of code style and we have tools that will automate this step for you.
+We enforce a minimum level of code style with our continuous intergration (the name is ``sunpy.sunpy (python_codestyle [linux]``).
+This runs a tool called `pre-commit <https://pre-commit.com/>`__.
 
-First step is to install pre-commit::
+Instead of installing this, you can use "tox" (which is used to run the sunpy test suite) to run these tools without having to setup anything within your own Python virtual environment::
+
+    $ tox -e codestyle
+
```

Comment:
After this command I would put an explanation of what will happen when you run this, i.e. it will make changes in the source to auto fix some issue etc (i.e. the bit lower down).

I would then move the install pre-commit instructions down under the "automate this" section?

Reply:
Added.